content/millennium/dstu2/infrastructure/document-reference.md

```diff
@@ -174,7 +174,7 @@ List an individual DocumentReference by its id:
 _Implementation Notes_
 
 * Contents of the document are found by following the Attachment URL. See more information on the [Binary resource] to determine what Authorization scopes are required, and how to set the Accept header when downloading document contents.
-
```

Comment:
add back this new line

Reply:
Done https://github.com/cerner/fhir.cerner.com/pull/233/commits/b285fd446ef7c6d7025915d48980e81777060efd#diff-b5f0beeb0c76f04f4fcbe824eb832abeR178